client/components/MostPopular.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import axios from 'axios';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchProducts } from '../store';
+
+const mapStateToProps = state => {
+  const { products } = state.product;
+  return {
+    products,
+  };
+};
+
+const mapDispatchToProps = dispatch => {
+  return {
+    fetchProducts: () => dispatch(fetchProducts()),
+  };
+};
+
+class MostPopular extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      orderItems: [],
+      mostPopular: [],
+    };
+  }
+
+  componentDidMount() {
+    const { orderItems } = this.state;
+    fetchProducts();
+    axios
```

Comment:
This should be a thunk.  Leave for now.  
(Update:  I think you are doing this because, you are using local state, correct?  If don't need local state, then can use thunk, otherwise ignore my comments.  Not all of them, just related to this!   :)  )
```suggestion
// TODO change to thunk
    axios
```